README.md

```diff
@@ -97,6 +98,33 @@ estraverse.traverse(tree, {
 });
 ```
 
+By passing `visitor.ignoreKeys` option along with `visitor.fallback`, you can skip specific keys in unknown nodes:
```

Comment:
Might be inconsistent voice in this documentation page. Previous paragraph (not part of changes in this PR) uses first person plural; these changes (consistent with ESLint docs) use second person. I don't know if that's important enough to require a change, but I wanted to call it out.